.github/workflows/tag-release.yml

```diff
@@ -216,3 +216,75 @@ jobs:
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
       - run: yarn release
+
+  # Deployment steps taken from https://github.com/colinwilson/static-site-to-vercel/blob/master/.github/workflows/deploy-preview.yml
+  repl_build:
+    name: Build REPL
+    runs-on: ubuntu-latest
+    needs: [unit_tests]
+    permissions:
+      deployments: write
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          cache: yarn
+          node-version: 20
+      - uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: stable
+          targets: wasm32-unknown-unknown
+      - name: Install wasm-opt
+        run: |
+          curl -L -O https://github.com/WebAssembly/binaryen/releases/download/version_116/binaryen-version_116-x86_64-linux.tar.gz
+          tar -xf binaryen-version_116-x86_64-linux.tar.gz
+          echo "$PWD/binaryen-version_116/bin" >> $GITHUB_PATH
+      - uses: Swatinem/rust-cache@v2
+        with:
+          key: wasm
+      - name: Bump max inotify watches
+        run: echo fs.inotify.max_user_watches=524288 | sudo tee -a /etc/sysctl.conf && sudo sysctl -p;
+      - run: yarn --frozen-lockfile
+      - name: Download @parcel/rust Linux Binaries artifact
+        uses: actions/download-artifact@v3
+        with:
+          name: Rust Linux Binaries
+          path: packages/core/rust
+      - run: yarn build
+      - run: yarn build-native-wasm
+      - run: yarn workspace @parcel/repl build
+      # - name: Upload REPL
+      #   uses: actions/upload-artifact@v3
+      #   with:
+      #     name: REPL
+      #     path: 'packages/dev/repl/dist'
+      - name: Start Deployment
+        uses: bobheadxi/deployments@v1
+        id: deployment
+        with:
+          step: start
+          token: ${{ secrets.GITHUB_TOKEN }}
+          ref: ${{ github.head_ref }}
+          env: Preview
```

Comment:
We should make that something like `Production`, though this isn't really exposed that much in the Github UI: https://github.com/parcel-bundler/parcel/deployments